libs/core/table/directives/table-container.directive.ts

```diff
@@ -0,0 +1,73 @@
+import {
+    ContentChild,
+    Directive,
+    ElementRef,
+    Input,
+    OnDestroy,
+    Renderer2,
+    booleanAttribute,
+    inject
+} from '@angular/core';
+import { ResizeObserverService } from '@fundamental-ngx/cdk/utils';
+import { FD_TOOLBAR } from '@fundamental-ngx/core/toolbar';
+import { Subscription } from 'rxjs';
+import { TableHeaderDirective } from './table-header.directive';
+
+@Directive({
+    selector: '[fdTableContainer], [fd-table-container]',
+    standalone: true
+})
+export class TableContainerDirective implements OnDestroy {
```

Comment:
I think if this is only used for the edge case of stickying the header when scrolling the outside container, the name should reflect that somehow, to avoid confusion for anyone that might stumble on this directive. With this name, you would think the table is always needing this container